src/public/parts/shells/chat/public/src/templates/world_info_chat_view.html

```
@@ -1,6 +1,8 @@
 <div class="card bg-base-100 shadow-xl mt-2 w-full" data-template-type="world">
 	<div class="card-body">
-		<img src="${avatar || 'https://api.iconify.design/line-md/map-marker-radius.svg'}" class="w-12 h-12 rounded-full mr-4" />
+		${avatar
+		? `<img src="${escapeHtml(avatar)}" class="w-12 h-12 rounded-full mr-4" svg-inliner-ignore />`
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)